vega_sim/quant/quant.py

```diff
@@ -0,0 +1,34 @@
+from math import log, sqrt, erf
+
+import vega_sim.proto.vega as vega_protos
+
+
+def probability_of_trading(
+    tau,
+    mu,
+    sigma,
+    lower_bound: float,
+    upper_bound: float,
+    best_price,
+    price,
+    side,
+):
+
+    stdev = sigma * sqrt(tau)
+    m = log(best_price) + (mu - 0.5 * sigma * sigma) * tau
+
+    if price < lower_bound or price > upper_bound:
+        return 0
+
+    min = cdf(m, stdev, lower_bound)
+    max = cdf(m, stdev, upper_bound)
```

Comment:
Should probably use scipy stats here rather than reimplementing locally

Reply:
Have switched over but am getting eeeever so slightly different probabilities from Lewis units tests.

- Scipy:  0.9997857769969356
- Local implementation: 0.9997857769969329

Am going to assume this is okay and update our unit tests to reflect the scipy values...?

Reply:
Yeah when the 15th significant figure is on our list of biggest accuracy issues I'll be happy :).

Side note: Are his/our unit tests comparing exact values of floats? They shouldn't be doing that in any case (for this exact reason)

Reply:
Floats, I have updated our tests a little so we just check all the limits.